config/keys.js

```diff
@@ -0,0 +1,8 @@
+// add this file to .gitignore
+
+module.exports = {
+        google: {
+            clientID: '558027906323-30k4oqhfe0m38iupmhef8p1hid76v6i5.apps.googleusercontent.com',
+            clientSecret: 'OzdguWo6g2TaUfrhOe8lPGse'
+        }
+};
```

Comment:
I don't think this should be pushed into stage. Update .gitignore appropriately to make sure this isn't being pushed to GitHub.

Will need to be removed from GitHub as well.